pkg/agent/agent_windows.go

```diff
@@ -218,6 +283,80 @@ func (i *Initializer) prepareOVSBridgeOnHNSNetwork() error {
 	return nil
 }
 
+func (i *Initializer) prepareOVSConfigForVM() error {
+	klog.InfoS("Performing OVS configuration", "hostIfName", i.nodeConfig.UplinkNetConfig.Name)
+	hostIfName := i.nodeConfig.UplinkNetConfig.Name
+	uplinkIfName := util.GenerateUplinkInterfaceName(hostIfName)
+	ovsPorts, ovsErr := i.ovsBridgeClient.GetPortList()
+	if ovsErr != nil {
+		klog.Errorf("Failed to list OVS ports")
+		return ovsErr
+	}
+	for index := range ovsPorts {
+		port := &ovsPorts[index]
+		if port.Name == hostIfName {
+			klog.Info("Uplink and host configuration exists in OVS")
```

Comment:
klog.Info("Uplink and host interface configuration exist in OVS")

Reply:
Done